src-docs/src/views/panel/panel_example.js

```diff
@@ -49,72 +88,111 @@ export const PanelExample = {
-          components like{' '}
-          <Link to="/layout/page">
-            <strong>EuiPage</strong>
-          </Link>{' '}
-          and{' '}
-          <Link to="/layout/popover">
-            <strong>EuiPopover</strong>
-          </Link>
-          .
-        </p>
+        <p>The most basic use is to simply add padding around your content.</p>
       ),
       props: { EuiPanel },
       snippet: panelSnippet,
       demo: <Panel />,
     },
     {
-      title: 'Panel can be hoverable',
+      title: 'Shadow',
       source: [
         {
           type: GuideSectionTypes.JS,
-          code: panelHoverSource,
+          code: panelShadowSource,
         },
         {
           type: GuideSectionTypes.HTML,
-          code: panelHoverHtml,
+          code: panelShadowHtml,
         },
       ],
       text: (
         <p>
-          Adding an <EuiCode>onClick</EuiCode> handler to the{' '}
-          <strong>EuiPanel</strong> will turn the wrapping element into a button
-          to allow for interaction.
+          <strong>EuiPanel</strong> can give depth to your container with{' '}
+          <EuiCode>hasShadow</EuiCode>.
         </p>
       ),
-      snippet: panelHoverSnippet,
-      demo: <PanelHover />,
+      props: { EuiPanel },
+      snippet: panelShadowSnippet,
+      demo: <PanelShadow />,
     },
     {
-      title: 'Panel beta badges',
+      title: 'Colors and corners',
       source: [
         {
           type: GuideSectionTypes.JS,
-          code: panelBadgeSource,
+          code: panelColorSource,
         },
         {
           type: GuideSectionTypes.HTML,
-          code: panelBadgeHtml,
+          code: panelColorHtml,
         },
       ],
       text: (
         <p>
-          Similar to{' '}
-          <Link to="/display/card">
-            <strong>EuiCard</strong>
-          </Link>
-          , panels can also accept an{' '}
-          <Link to="/display/badge">
-            <strong>EuiBetaBadge</strong>
-          </Link>
-          .
+          Use <EuiCode>color</EuiCode> to add background shading to your panel
+          and provide an additional helpful aesthetic to your container in
+          context. Be mindful to use color sparingly. You can also remove the
+          rounded corners depending on the placement of your panel with{' '}
+          <EuiCode language="tsx">{'borderRadius="none"'}</EuiCode>
         </p>
       ),
-      snippet: panelBadgeSnippet,
+      props: { EuiPanel },
+      snippet: panelColorSnippet,
+      demo: <PanelColor />,
+    },
+    {
+      title: 'Growing height',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: panelGrowSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: panelGrowHtml,
+        },
+      ],
+      text: (
+        <p>
+          Using <strong>EuiPanel</strong> in an{' '}
+          <Link to="/layout/flex">
```

Comment:
Now that we have the ability to deeplink should we direct this URL to a more specific section like `layout/flex#flex-item-can-individually-turn-off-stretching`

Reply:
Yeah I only looked for an EuiFlexItem specific section (there was none), but didn't think about the stretching section. 👍 And actually there is a section just above that specifically about this behavior as well so I'll link to that one.

```suggestion
          <Link to="/layout/flex#panels-grow-to-fill-flex-items">
```